### ISSUE TITLE
Errors after upgrading to 1.8

### ISSUE THREAD

**Issue by Haron-Prime** [archlinuxfr/yaourt#241]
After upgrading to 1.8-1 there were errors in the output yaourt -Syua

```
:: Обновление баз данных пакетов...
 core                     121,7 KiB  75,8K/s 00:02 [###################] 100%
 extra                   1768,1 KiB  82,1K/s 00:22 [###################] 100%
 community                  3,4 MiB   121K/s 00:28 [###################] 100%
 multilib                 162,5 KiB  79,3K/s 00:02 [###################] 100%
 archlinuxfr               11,2 KiB  3,64M/s 00:00 [###################] 100%
  dupeguru-se : Нет ответственного 
  ssx : Нет ответственного 
 Другие пакеты: - 114 / 114
ошибка: file owned by 'pacmanxg4' and 'pacmanxg4-bin': 'opt/Almin-Soft/pacmanxg/dizz.sh'
ошибка: file owned by 'pacmanxg4' and 'pacmanxg4-bin': 'opt/Almin-Soft/pacmanxg/langs/ca.lang'
ошибка: file owned by 'pacmanxg4' and 'pacmanxg4-bin': 'opt/Almin-Soft/pacmanxg/langs/de.lang'
ошибка: file owned by 'pacmanxg4' and 'pacmanxg4-bin': 'opt/Almin-Soft/pacmanxg/langs/el.lang'
ошибка: file owned by 'pacmanxg4' and 'pacmanxg4-bin': 'opt/Almin-Soft/pacmanxg/langs/en.lang'
ошибка: file owned by 'pacmanxg4' and 'pacmanxg4-bin': 'opt/Almin-Soft/pacmanxg/langs/es.lang'
ошибка: file owned by 'pacmanxg4' and 'pacmanxg4-bin': 'opt/Almin-Soft/pacmanxg/langs/fr.lang'
ошибка: file owned by 'pacmanxg4' and 'pacmanxg4-bin': 'opt/Almin-Soft/pacmanxg/langs/gl.lang'
ошибка: file owned by 'pacmanxg4' and 'pacmanxg4-bin': 'opt/Almin-Soft/pacmanxg/langs/pl.lang'
ошибка: file owned by 'pacmanxg4' and 'pacmanxg4-bin': 'opt/Almin-Soft/pacmanxg/langs/pt-br.lang'
ошибка: file owned by 'pacmanxg4' and 'pacmanxg4-bin': 'opt/Almin-Soft/pacmanxg/langs/ru.lang'
ошибка: file owned by 'pacmanxg4' and 'pacmanxg4-bin': 'opt/Almin-Soft/pacmanxg/pacmanxg'
ошибка: file owned by 'pacmanxg4' and 'pacmanxg4-bin': 'opt/Almin-Soft/pacmanxg/repacman'
ошибка: file owned by 'pacmanxg4' and 'pacmanxg4-bin': 'opt/Almin-Soft/themes/Graybird-like (Bridge)'
ошибка: file owned by 'pacmanxg4' and 'pacmanxg4-bin': 'opt/Almin-Soft/themes/Menda-like (Manjaro 0.10)'
ошибка: file owned by 'pacmanxg4' and 'pacmanxg4-bin': 'opt/Almin-Soft/themes/Munix Frost Light (Antergos)'
ошибка: file owned by 'pacmanxg4' and 'pacmanxg4-bin': 'opt/Almin-Soft/themes/Oxigen-like (Netrunner)'
ошибка: file owned by 'pacmanxg4' and 'pacmanxg4-bin': 'opt/Almin-Soft/themes/dusk (Chakra)'
ошибка: file owned by 'pacmanxg4' and 'pacmanxg4-bin': 'usr/share/applications/pacmanxg.desktop'
ошибка: file owned by 'pacmanxg4' and 'pacmanxg4-bin': 'usr/share/pixmaps/pacmanxg.png'
```

If downgrade to version 1.7-1, then there are no errors
```
uname -a
Linux arch 4.4.3-1-ARCH #1 SMP PREEMPT Fri Feb 26 15:09:29 CET 2016 x86_64 GNU/Linux
```
yaourt 1.8-1
package-query 1.8-1

P.S> 
Errors are present, even if completely removed pacmanxg4

**Comment by larchunix** [comment]
What is the output of `pacman -Dk` ?

**Comment by Haron-Prime** [comment]
```
pacman -Dk     
ошибка: file owned by 'pacmanxg4' and 'pacmanxg4-bin': 'opt/Almin-Soft/pacmanxg/dizz.sh'
ошибка: file owned by 'pacmanxg4' and 'pacmanxg4-bin': 'opt/Almin-Soft/pacmanxg/langs/ca.lang'
ошибка: file owned by 'pacmanxg4' and 'pacmanxg4-bin': 'opt/Almin-Soft/pacmanxg/langs/de.lang'
ошибка: file owned by 'pacmanxg4' and 'pacmanxg4-bin': 'opt/Almin-Soft/pacmanxg/langs/el.lang'
ошибка: file owned by 'pacmanxg4' and 'pacmanxg4-bin': 'opt/Almin-Soft/pacmanxg/langs/en.lang'
ошибка: file owned by 'pacmanxg4' and 'pacmanxg4-bin': 'opt/Almin-Soft/pacmanxg/langs/es.lang'
ошибка: file owned by 'pacmanxg4' and 'pacmanxg4-bin': 'opt/Almin-Soft/pacmanxg/langs/fr.lang'
ошибка: file owned by 'pacmanxg4' and 'pacmanxg4-bin': 'opt/Almin-Soft/pacmanxg/langs/gl.lang'
ошибка: file owned by 'pacmanxg4' and 'pacmanxg4-bin': 'opt/Almin-Soft/pacmanxg/langs/pl.lang'
ошибка: file owned by 'pacmanxg4' and 'pacmanxg4-bin': 'opt/Almin-Soft/pacmanxg/langs/pt-br.lang'
ошибка: file owned by 'pacmanxg4' and 'pacmanxg4-bin': 'opt/Almin-Soft/pacmanxg/langs/ru.lang'
ошибка: file owned by 'pacmanxg4' and 'pacmanxg4-bin': 'opt/Almin-Soft/pacmanxg/pacmanxg'
ошибка: file owned by 'pacmanxg4' and 'pacmanxg4-bin': 'opt/Almin-Soft/pacmanxg/repacman'
ошибка: file owned by 'pacmanxg4' and 'pacmanxg4-bin': 'opt/Almin-Soft/themes/Graybird-like (Bridge)'
ошибка: file owned by 'pacmanxg4' and 'pacmanxg4-bin': 'opt/Almin-Soft/themes/Menda-like (Manjaro 0.10)'
ошибка: file owned by 'pacmanxg4' and 'pacmanxg4-bin': 'opt/Almin-Soft/themes/Munix Frost Light (Antergos)'
ошибка: file owned by 'pacmanxg4' and 'pacmanxg4-bin': 'opt/Almin-Soft/themes/Oxigen-like (Netrunner)'
ошибка: file owned by 'pacmanxg4' and 'pacmanxg4-bin': 'opt/Almin-Soft/themes/dusk (Chakra)'
ошибка: file owned by 'pacmanxg4' and 'pacmanxg4-bin': 'usr/share/applications/pacmanxg.desktop'
ошибка: file owned by 'pacmanxg4' and 'pacmanxg4-bin': 'usr/share/pixmaps/pacmanxg.png'
```

**Comment by larchunix** [comment]
This means it is not a `yaourt` issue. Local DB check was broken in `yaourt 1.7` and has been fixed in version `1.8`.

You can bypass the check by setting `NO_TESTDB=1` in `yaourtrc` but IMHO the right solution is to fix your local package database.

**Comment by Haron-Prime** [comment]
Thank you. I figured out the problem.
